validation-status.md

```diff
@@ -0,0 +1,565 @@
+# Maturity of these profiles
+## Validation by HL7 Netherlands
+Nictiz develops FHIR profiles (and related conformance resources) representing the zibs. These profiles undergo a validation process at HL7 Netherlands[^1], which checks them on aspects like alignment with international FHIR usage, re-usability, usage of terminology, etc. If one or more of these aspects are deemed incorrect, HL7 Netherlands asks Nictiz to adjust the profile accordingly. Only if all these aspects are deemed correct by HL7 Netherlands, the profile passes validation.
+
+In terms of the NEN7522, HL7 Netherlands fulfills the roles of "expert" and "autorisator".
+

```

Comment:
Shall we provide an overview here of the status values?

- not submitted for validation
- in progress
- validated 
- rejected

If something is rejected, the next step would be to update it to in progress once we deliver updated profiles, correct?